src/pages/components/Demo/components/Translate.tsx

```diff
@@ -8,27 +19,41 @@ const Translate = () => {
   const [isPredictLoading, setIsPredictLoading] = useState(false);
   const [translation, setTranslation] = useState('');
   const [text, setText] = useState('');
+  const [languagePair, setLanguagePair] = useState({ from: 'Igbo', to: 'English' });
 
   const handleTranslate = async () => {
     try {
       setIsPredictLoading(true);
-      const fetchedTranslation = await postTranslationEndpoint({ text });
+      const fetchedTranslation = await postTranslationEndpoint({ text, languagePair });
       setTranslation(fetchedTranslation.translation);
     } finally {
       setIsPredictLoading(false);
     }
   };
 
+  const switchLanguagePair = () => {
+    setLanguagePair({
+      from: languagePair.to,
+      to: languagePair.from,
+    });
+  };
+
   return (
     <Box maxWidth="1200px" width="full">
       <VStack width="full" p={4} gap={4}>
-        <HStack flexDirection={{ base: 'column', md: 'row' }} width="full" gap={6} display="flex">
+        <HStack
+          flexDirection={{ base: 'column', md: 'row' }}
+          width="full"
+          gap={6}
+          display="flex"
+          position="relative"
+        >
           <VStack width="full" textAlign="start">
             <Heading color="gray.800" fontSize="xl" width="full" textAlign="center">
-              Igbo
+              {languagePair.from}
```

Comment:
Ah, I see why you wanted to do `languagePair = { from: 'Igbo', to: 'English' }`.

We still want to use `LanguageEnum` above, but we also now want to create a map that maps from the `LanguageEnum` to its corresponding label. Creating this map makes it so that if we wanted to change the UI representation of the enum value, we don't need to change the actual enum value and just the label.

So that would look something like this:

```ts
const LanguageEnumLabels: {
  [key in LanguageEnum]: {
    label: string,
  }
} = {
  [LanguageEnum.IGBO]: {
    label: 'Igbo'
  },
  [LanguageEnum.English]: {
    label: 'English'
  }
}

...
<Heading>
  {LanguageEnumLabels[languagePair.from].label}
</Heading>
```